tests/RequestBuilderTest.php

```diff
@@ -9,37 +9,151 @@
 
 namespace Mailgun\Tests;
 
+use Http\Message\MultipartStream\MultipartStreamBuilder;
+use Http\Message\RequestFactory;
 use Mailgun\RequestBuilder;
+use PHPUnit\Framework\TestCase;
```

Comment:
But there is
https://github.com/mailgun/mailgun-php/blob/master/composer.json#L14